rotkehlchen/chain/ethereum/airdrops.py

```diff
@@ -386,6 +388,8 @@ def check_airdrops(
                     'link': airdrop_data.url,
                     'claimed': False,
                 }
+                if protocol_name in airdrops_without_decoder:
+                    temp_found_data[addr][protocol_name]['missing_decoder'] = True  # type: ignore
```

Comment:
not for this PR but we probably should change the way data is gathered here. We are repeating `asset`, `link` and now `missing_decoder` for each address of a given airdrop